### ISSUE TITLE
We are running rc11 and we are seeing duplicate processes spawned after starting a child.

### ISSUE THREAD

**Issue by QuantamHD** [derekkraan/horde#88]
So here's as much information as I have gathered so far.

* This issue did not appear at rc6 we have seen it at rc10, and rc11.
* The process in question is a transient process
* It is named using horde; with the via tuple `{:via, Horde.Registry, {Engine.HordeRegistry, {__MODULE__, struct.action_id}}}
`
* startup code

```elixir
      child_spec =
        {WatchFolderRunner, {workflow_id, watch_folder}}
        |> Supervisor.child_spec(id: "#{WatchFolderRunner}_#{watch_folder.action_id}")

      Horde.Supervisor.start_child(
        Engine.HordeSupervisor,
        child_spec
      )
```

* The problem only occurs in multinode setups specifically we run 6. The problem happens very frequently. 
* Usually what happens is that one process will be started on the node that calls the above code and another version will be started on a remote node.

### Opinion Section

I think this change may have been introduced around rc7 or rc8 with the change to one CRDT. I have not validated it though.

**Comment by derekkraan** [comment]
Could be related to #90. Do you use `set_members/2`?

**Comment by derekkraan** [comment]
Specifically, this could happen if the remote node thinks the node the process has been started on is dead.

**Comment by QuantamHD** [comment]
We do use set_members/2 at initialization

**Comment by derekkraan** [comment]
Hi @QuantamHD, could you test this on rc12 and let me know if you're still seeing this problem?

**Comment by derekkraan** [comment]
Hi @QuantamHD, is this still an issue in `rc.12`?

**Comment by derekkraan** [comment]
I have released 0.5.0, and will close this issue. Feel free to re-open in case it has not in fact been solved in recent versions.
